roles/kubernetes/master/tasks/encrypt-at-rest.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Write secrets for encrypting secret data at rest
+  template:
+    src: secrets.yaml.j2
```

Comment:
How about rename this file to `secrets_encryption.yaml.j2` ?

Reply:
Sure. Let me fix that.